docs/reference/snapshot-restore/apis/restore-snapshot-api.asciidoc

```diff
@@ -144,6 +144,10 @@ backing indices are configured using the data stream's matching index template.
 If `true`, the request restores aliases for any restored data streams and
 indices. If `false`, the request doesn't restore aliases. Defaults to `true`.
 
+`silent`::
+(Optional, Boolean)
+If `true`, logging happens at a lower level. Defaults to `false`.
```

Comment:
Expression is got from the similar "silent" parameter of https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java#L416 . Not sure if it's super useful, as it does not mention what the lower level is (DEBUG or INFO). But I guessed the original description did not want to make promises about the log level, and that's why I also adopted a similar description. Feedback welcome!